resources/assets/js/videos/components/videoScreen.vue

```diff
@@ -452,10 +452,18 @@ export default {
         extractAnnotationFromFeature(feature) {
             return feature.get('annotation');
         },
-        handleFeatureSelect(e) {
+        handleFeatureSelect() {
+            // New selected annotations
+            // Don't use event.selected here because then shift select won't work in current frame.
+            let selected = this.selectInteraction.getFeatures().getArray().map(this.extractAnnotationFromFeature);
+            // Old selected annotations
+            // Don't use event.deselected here because it uses selections in current frame only,
+            // see https://github.com/biigle/core/issues/552.
+            let deselected = this.annotations.filter(a => a.selected)
```

Comment:
```suggestion
            let deselected = this.annotations.filter(a => a.isSelected)
```

This must be `isSelected` because `selected` stores the time if the annotation is selected and the time could be `0` which would be false here. I should rather have called it `selectedTime` :man_shrugging: